### PR TITLE
ISS-6:Add prompts for showing patients on INH, WHO Stage and TB Rx

### PR DESCRIPTION
**Pull request by reagan-meant**

### CODE REVIEW

File: omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-EncounterPage.xml [hunk 1]
```diff
@@ -301,6 +301,16 @@
             });
 
             jq(document).ready(function () {
+
+                /* Prompt if client is on TB Rx or completed it */
+                const lastTBStatusValue = jq("#lastTBStatus").html().split('#');
```
Comment (reagan-meant) [inline omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-EncounterPage.xml]: I couldnt get a covenient way to get the conceptId of lastTBStatus...I tried <span style="display: none" id="lastTBStatus"><lookup expression="fn.latestObs('90216').valueCoded.concetId"/></span> but it doesnt work.
This is why I manipulated the output

**Comment by ssmusoke** [comment]
@reagan-meant There is a merge conflict here please fix and add a link to the Asana card too

**Comment by reagan-meant** [comment]
https://app.asana.com/0/1164864596604507/1164867478655791 
@ssmusoke

File: omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-EncounterPage.xml (hunk 1 above)
Comment (ssmusoke) [reply omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-EncounterPage.xml]: @reagan-meant use getValue('fieldname-id')

**Pull request merged**